node/pkg/watchers/evm/watcher.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/wormhole-foundation/wormhole/sdk/vaa"
 )
 
+const (
+	magicNumberPublishImmediately = uint8(200)
```

Comment:
Can we please put constants like this in the sdk rather than having them spread out in various files?

Reply:
Done.